specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/LinkedService.json

```diff
@@ -571,6 +571,10 @@
         "encryptedCredential": {
           "type": "object",
           "description": "The encrypted credential used for authentication. Credentials are encrypted using the integration runtime credential manager. Type: string (or Expression with resultType string)."
+        },
+        "isEspEnabled": {
+          "type": "object",
```

Comment:
Curious why this is modeled as an `object` if the type is boolean?

Reply:
Because object allows user to put some expressions to indicate a boolean value.